dasgoclient.spec

```diff
@@ -44,3 +44,13 @@ if [ "`ls ${RPM_INSTALL_PREFIX}/*/%{pkgcategory}/%{pkgname}/v*/etc/profile.d/ini
   /bin/cp -f ${RPM_INSTALL_PREFIX}/%{pkgrel}/etc/dasgoclient $RPM_INSTALL_PREFIX/common/dasgoclient.tmp
   mv $RPM_INSTALL_PREFIX/common/dasgoclient.tmp $RPM_INSTALL_PREFIX/common/dasgoclient
 fi
+
+# make soft link between dasgoclient and das_client
+if [ -f $RPM_INSTALL_PREFIX/common/das_client ]; then
+   rm $RPM_INSTALL_PREFIX/common/das_client
+fi
+ln -s $RPM_INSTALL_PREFIX/common/dasgoclient $RPM_INSTALL_PREFIX/common/das_client
```

Comment:
@vkuznet , one problem which we will have is that if someone install an old cmssw release after installing this? old release will bring an old das_client which will override the das_client symlink. I would suggest to drop the above change and make use of override/bin/das_client. No package yet creates share/overrides/bin/das_client so I would suggest to create override/bin/das_client symlink pointing to common/dasgocleint.

Only issue left is for those release where we call das_client.py (which is already in available in share/override/bin) and I do not see any simple solution for this.